.releaserc.json

```diff
@@ -0,0 +1,21 @@
+{
+  "branches": ["semantic-release"],
```

Comment:
At the end, the release branch should be set to `dev` as it is the default branch the PR are merged into